lib/network/probe/HealthProbeServer.js

```diff
@@ -0,0 +1,99 @@
+const httpServer = require('../http/server');
```

Comment:
I left a comment on your backbeat PR about returning errors instead of a generic "Service Unavailable" response to help with diagnosing. I wanted to also mention the lack of JsDocs and there is a bit of repetition in here we could also refactor if we wanted to. Alternatively, we could create a more generic `ProbeServer` for all the simple probelike requests ( liveness, ready, metrics, startup, etc ) and slowly transition.

But of course, we could just ship it... :boat: